diff --git a/config/environments/test.rb b/config/environments/test.rb
index 0d1b42e..0a76cb1 100644
--- a/config/environments/test.rb
+++ b/config/environments/test.rb
@@ -24,6 +24,6 @@ config.action_controller.session = {
 # Skip protect_from_forgery in requests http://m.onkey.org/2007/9/28/csrf-protection-for-your-existing-rails-application
 config.action_controller.allow_forgery_protection  = false
 
-config.gem "thoughtbot-shoulda", :lib => "shoulda", :source => "http://gems.github.com"
-config.gem "nofxx-object_daddy", :lib => "object_daddy", :source => "http://gems.github.com"
+config.gem "shoulda", :lib => "shoulda", :source => "http://gems.github.com"
+config.gem "object_daddy", :lib => "object_daddy", :source => "http://gems.github.com"
 config.gem "mocha"
diff --git a/config/environments/test_pgsql.rb b/config/environments/test_pgsql.rb
index 0d1b42e..0a76cb1 100644
--- a/config/environments/test_pgsql.rb
+++ b/config/environments/test_pgsql.rb
@@ -24,6 +24,6 @@ config.action_controller.session = {
 # Skip protect_from_forgery in requests http://m.onkey.org/2007/9/28/csrf-protection-for-your-existing-rails-application
 config.action_controller.allow_forgery_protection  = false
 
-config.gem "thoughtbot-shoulda", :lib => "shoulda", :source => "http://gems.github.com"
-config.gem "nofxx-object_daddy", :lib => "object_daddy", :source => "http://gems.github.com"
+config.gem "shoulda", :lib => "shoulda", :source => "http://gems.github.com"
+config.gem "object_daddy", :lib => "object_daddy", :source => "http://gems.github.com"
 config.gem "mocha"
diff --git a/config/environments/test_sqlite3.rb b/config/environments/test_sqlite3.rb
index 0d1b42e..0a76cb1 100644
--- a/config/environments/test_sqlite3.rb
+++ b/config/environments/test_sqlite3.rb
@@ -24,6 +24,6 @@ config.action_controller.session = {
 # Skip protect_from_forgery in requests http://m.onkey.org/2007/9/28/csrf-protection-for-your-existing-rails-application
 config.action_controller.allow_forgery_protection  = false
 
-config.gem "thoughtbot-shoulda", :lib => "shoulda", :source => "http://gems.github.com"
-config.gem "nofxx-object_daddy", :lib => "object_daddy", :source => "http://gems.github.com"
+config.gem "shoulda", :lib => "shoulda", :source => "http://gems.github.com"
+config.gem "object_daddy", :lib => "object_daddy", :source => "http://gems.github.com"
 config.gem "mocha"
