diff --git a/config/environments/test.rb b/config/environments/test.rb
index 291e80c..b1d12ff 100644
--- a/config/environments/test.rb
+++ b/config/environments/test.rb
@@ -24,6 +24,6 @@ config.action_controller.session = {
 # Skip protect_from_forgery in requests http://m.onkey.org/2007/9/28/csrf-protection-for-your-existing-rails-application
 config.action_controller.allow_forgery_protection  = false
 
-config.gem "thoughtbot-shoulda", :lib => "shoulda", :source => "http://gems.github.com"
-config.gem "edavis10-object_daddy", :lib => "object_daddy"
+config.gem "shoulda"
+config.gem "object_daddy"
 config.gem "mocha"
